bom-internal/pom.xml

```diff
@@ -37,7 +37,7 @@ POM as their parent.
 
     <groupId>io.dockstore</groupId>
     <artifactId>bom-internal</artifactId>
-    <version>1.11.0-alpha.0-SNAPSHOT</version>
```

Comment:
Doesn't seem right for going into develop, master should get 1.10.2-SNAPSHOT if anything, develop should keep 1.11.0-alpha.0-SNAPSHOT

Reply:
Agreed, but it it's still on 1.10.

Reply:
Interesting, think the Maven versions plugin file is missing this one